### PR TITLE
Fix up package configuration docs

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details.

### CODE REVIEW

File: docs/config-file-package-configuration-yml.md [hunk 1]
```diff
@@ -1,38 +1,43 @@
-# The package configuration file (Experimental)
+# The Package Configuration File (Experimental)
 
-A package configuration sets up path excludes and license finding curations specific to a package (dependency) and 
-provenance. It is thus a similar concept to the `.ort.yml` file which allows to do similar configuration for projects as
-opposed to packages.
+A package configuration file allows you to define path excludes and license finding curations for a specific package (dependency) and 
+provenance. Conceptually, the file is similar to [.ort.yml](https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-ort-yml.md), but it is used only for packages included via a package manager as project dependencies, and not for the project's own source code respository to be scanned.
```
Comment (fviernau) [inline docs/config-file-package-configuration-yml.md:4]: `only for packages`: is it actually now (still) clear that the ort.yml configures projects while the pkg config configures packages?

File: docs/config-file-package-configuration-yml.md [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
 
-## Specifying the target package and provenance
+### When To Use
+Use a package configuration file to:
+
+-  mark files and directories as not included in released artifacts -- use it to make clear that license findings in documentation or tests in a package sources do not apply to the release (binary) artifact which is a dependency in your project
+-  overwrite scanner findings to correct identified license in a dependency for a specific file or directory
```
Comment (fviernau) [inline docs/config-file-package-configuration-yml.md:10]: overwrite -> override?

Comment (fviernau) [reply docs/config-file-package-configuration-yml.md:10]: identified license -> identified licenses

Comment (fviernau) [reply docs/config-file-package-configuration-yml.md:10]: remove 'or director' as this is not how it (currently) works.

File: docs/config-file-package-configuration-yml.md [hunk 3]
```diff
@@ -1,38 +1,43 @@
...
-## Specifying path excludes and license finding curations
+## Defining Path Excludes and License Finding Curations
 
-Path excludes are used to tell ORT that license findings in certain files can be ignored, for example because they
-belong to the documentation of the package. License finding curations are used to fix wrong scan results, for example
-if a wrong license was detected, or if a finding is a false positive. The entries for path excludes and license finding
-curations have the same syntax and semantic as the analog entries for the `ort.yml`, see
-[Excluding paths](config-file-ort-yml.md#excluding-paths) and
-[Curating license findings](config-file-ort-yml.md#curating-license-findings).
+Path excludes define which code is not part of the distributed release artifacts for a package, for example code found in the source repository but only used for building, documenting or testing the code. License finding curations are used to fix incorrect scan results, for example if a wrong license was detected, or if a finding is a false positive.
```
Comment (fviernau) [inline docs/config-file-package-configuration-yml.md:36]: artifacts -> artifact

File: docs/config-file-package-configuration-yml.md [hunk 4]
```diff
@@ -51,24 +56,69 @@ curations have the same syntax and semantic as the analog entries for the `ort.y
...
-## Organizing package configurations
+## Command Line
 
-ORT currently provides two alternatives for organizing package configurations to choose from. It is required for both
-options that there is at most one package configuration provided for any (package, provenance) combination.
+ORT offers two different ways to use package configurations:
 
-### Configuration directory
+- A single configuration `.yml` containing an array with each entry defining the configuration for one package
+- A directory with configuration files for each `package id`, with one file for each configured package/provenance combination
```
Comment (fviernau) [inline docs/config-file-package-configuration-yml.md:64]: each `package id` VS. each configured package/provenance combination

..is contradictory. the latter is the right one.

File: docs/config-file-package-configuration-yml.md [hunk 5]
```diff
@@ -51,24 +56,69 @@ curations have the same syntax and semantic as the analog entries for the `ort.y
...
- 
-### Configuration file
+Note that in both of the above options only one package configuration can exist for a package/provenance combination.
+
+### Using a Package Configuration Directory
 
-A single `.yml` containing one large array with each entry being one package configuration as shown in the parent
-section. This file can be specified via the `--package-configuration-file` option to the *evaluator* and the *reporter*
-command respectively. For example such single file would look as follows:
+To use a directory with configuration files for each `package id`, pass it to the `--package-configuration-dir` option of the _evaluator_:
```
Comment (fviernau) [inline docs/config-file-package-configuration-yml.md:70]: again: for each package/provenance

File: docs/config-file-package-configuration-yml.md [hunk 6]
```diff
@@ -51,24 +56,69 @@ curations have the same syntax and semantic as the analog entries for the `ort.y
...
+
+```bash
+cli/build/install/ort/bin/ort evaluate
+  -i [scanner-output-path]/scan-result.yml
+  -o [evaluator-output-path]/evaluator-result.yml
+  --output-formats YAML
+  --license-configuration-file [ort-configuration-path]/licenses.yml
+  --package-curations-file [ort-configuration-path]/curations.yml
+  --package-configuration-dir [ort-configuration-path]/packages
```
Comment (fviernau) [inline docs/config-file-package-configuration-yml.md:79]: is hit possible to highligt this e.g. via red color?

File: docs/config-file-package-configuration-yml.md (hunk 2 above)
Comment (tsteenbe) [reply docs/config-file-package-configuration-yml.md:10]: > overwrite -> override?

"Override" is the cancellation of some previous action or decision. "Overwrite" specifically refers to something being written over something previously written. In other words, actions and decisions are overriden. Information is overwritten. Furthermore it is overwritten with new information.

Source: https://english.stackexchange.com/questions/88400/when-to-use-override-and-overwrite

File: docs/config-file-package-configuration-yml.md (hunk 3 above)
Comment (tsteenbe) [reply docs/config-file-package-configuration-yml.md:36]: One package can have multiple release artifact, changed it to "part of the distributed release artifact(s) for a package"

File: docs/config-file-package-configuration-yml.md (hunk 4 above)
Comment (tsteenbe) [reply docs/config-file-package-configuration-yml.md:64]: Fixed

File: docs/config-file-package-configuration-yml.md (hunk 6 above)
Comment (tsteenbe) [reply docs/config-file-package-configuration-yml.md:79]: No, we are using (GitHub flavored) Markdown for our docs and Markdown is very simple so no highlighting within codeblocks

**Pull request merged**